programs/drift/src/math/margin.rs

```diff
@@ -236,6 +238,51 @@ pub fn calculate_user_safest_position_tiers(
     Ok((safest_tier_spot_liablity, safest_tier_perp_liablity))
 }
 
+pub fn calculate_perp_fuel_bonus(
+    perp_market: &PerpMarket,
+    base_asset_value: i128,
+    fuel_bonus_numerator: i64, // fuel_perp_delta: Option<(u16, i64)>
+) -> DriftResult<u64> {
+    let result: u64 = if base_asset_value.unsigned_abs() < 1 {
```

Comment:
just check if equal to 0? lol

Reply:
wanted to save compute for small stuff thats negligible (was thinking of a higher limit tbh)

Reply:
but `< 1` is the same as `= 0` so you dont save anything here?